src/webrtc/call.ts

```diff
@@ -267,6 +273,34 @@ const CALL_TIMEOUT_MS = 60 * 1000; // ms
 const CALL_LENGTH_INTERVAL = 1000; // ms
 /** The time after which we end the call, if ICE got disconnected */
 const ICE_DISCONNECTED_TIMEOUT = 30 * 1000; // ms
+/**
+ * The time we wait for call feed size and visibility changing before we send a
+ * new m.call.track_subscription
+ */
+const SUBSCRIBE_TO_FOCUS_TIMEOUT = 2 * 1000;
+
+const SIMULCAST_ENCODINGS = [
+    // Order is important here: some browsers (e.g.
+    // Chrome) will only send some of the encodings, if
+    // the track has a resolution to low for it to send
+    // all, in that case the encoding higher in the list
+    // has priority and therefore we put full as first
+    // as we always want to send the full resolution
+    {
+        maxBitrate: 4_500_000,
+        rid: SimulcastResolution.Full,
+    },
+    {
+        maxBitrate: 1_500_000,
+        rid: SimulcastResolution.Half,
+        scaleResolutionDownBy: 2.0,
+    },
+    {
+        maxBitrate: 300_000,
+        rid: SimulcastResolution.Quarter,
+        scaleResolutionDownBy: 4.0,
+    },
+];
```

Comment:
Just to note that the bitrates here are probably defined assuming 720p (ie. what we ask for in getUserMedia) but screenshares will be different resolutions, so these figures might not be as sensible for those (plus of course we may not actually get 720p).

Reply:
So I think they are actually not necessary and we should be fine without setting them - let's see :crossed_fingers: